user_manual/groupware/calendar.rst

```diff
@@ -179,6 +179,12 @@ If you want to edit advanced details such as the **Attendees** or **Reminders**,
 want to set the event as a repeating-event, click on the ``More`` button to open the advanced
 sidebar editor.
 
+Add Talk conversation
+~~~~~~~~~~~~~~~~~~~~~
+You can include an existing Talk conversation in your event by clicking "Add Talk conversation". To view the list of existing Talk conversations, ensure the Talk app is enabled. If you'd like to create a new Talk conversation, you can do so directly from the same modal.
+
+.. figure:: images/add-talk-room.png
+
 .. note:: If you always want to open the advanced sidebar editor instead of the
```

Comment:
The note underneath belongs to the "Create a new event" section, your addition should probably be moved further down

Reply:
I did that on purpose because add talk conversation is part of the create new event as a process :) and you can only access it after you open the side bar.